packages/suite-desktop-core/e2e/support/pageActions/settingsActions.ts

```diff
@@ -212,13 +211,4 @@ export class SettingsActions {
             await expect(this.notificationSuccessToast).toBeVisible();
         });
     }
-
```

Comment:
moved to trezorInputActions (wordInputAction) since the action fall to same category of specific input to trezor like pin, seed etc